packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -813,22 +834,20 @@ describe('src/cy/commands/actions/click', () => {
     })
 
     it('places cursor at the end of [contenteditable]', () => {
-      cy.get('[contenteditable]:first')
-      .invoke('html', '<div><br></div>').click()
-      .then(expectCaret(0))
+      cy.get('[contenteditable]:first').as('edit')
```

Comment:
Another test fallen victim to the breaking change around cy.invoke; because it's now a query, it can (will) be called multiple times while executing other commands; this resulted in the cursor moving around and the test failing.

Fortunately, the solution is simple; move .click() to its own chain, so .invoke() isn't part of its subject. I'll include this recommendation as part of the patch notes / upgrade guide when Detached DOM goes out.